io.github.tralph3.Steam_Metadata_Editor.yml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.tralph3.Steam_Metadata_Editor
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: start.sh
+finish-args:
+  - --filesystem=home:rw
```

Comment:
Any reason it needs full filesystem acess instead of only Steam directories? `~/.var/app/com.valvesoftware.Steam/.local/share/Steam` and `xdg-data/Steam` should suffice.

Reply:
I said why in the first comment, at the end of the checklist. Neither of those directories hold the default steam install. It's in `~/.local/share/Steam/`.

Reply:
~/.local/share is xdg-data.

Reply:
Oh, and I assume the other one is the path for the flatpak version of Steam? Then maybe yeah.

Can you write it like this?:

    - --filesystem=xdg-data/Steam:rw

And it works or do I have to specify `~/.local/share`?

Reply:
Ok, I verified that it does work.